app/config/config.yml

```diff
@@ -36,7 +36,7 @@ framework:
     csrf_protection: ~
     validation:      { enable_annotations: true }
     templating:
-        engines: ['twig']
+        engines: ['twig','php']
```

Comment:
well, this has been resolved in the bundle, so there is no need to add it anymore. Enabling it was a workaround for the KnpPaginatorBundle bug